ee/codegen/src/__test__/workflow-sandbox.test.ts

```diff
@@ -430,5 +430,165 @@ describe("Workflow Sandbox", () => {
 
       expect(result).toMatchSnapshot();
     });
+
+    it("should handle mocks with undefined then_outputs without throwing TypeError", async () => {
+      /**
+       * Tests that mocks with undefined then_outputs don't throw TypeError when
+       * Object.entries is called on them.
+       */
+
+      const writer = new Writer();
+      const uniqueWorkflowContext = workflowContextFactory();
+      const inputVariable: VellumVariable = {
+        id: "1",
+        key: "test_input",
+        type: "STRING",
+      };
+
+      uniqueWorkflowContext.addInputVariableContext(
+        inputVariableContextFactory({
+          inputVariableData: inputVariable,
+          workflowContext: uniqueWorkflowContext,
+        })
+      );
+
+      const genericNodeData = genericNodeFactory();
+      await nodeContextFactory({
+        workflowContext: uniqueWorkflowContext,
+        nodeData: genericNodeData,
+      });
+
+      // GIVEN a mock with undefined then_outputs
+      const sandboxInputs: WorkflowSandboxDatasetRow[] = [
+        {
+          label: "Scenario with Mock without then_outputs",
+          inputs: [
+            {
+              name: inputVariable.key,
+              type: "STRING",
+              value: "test-value",
+            },
+          ],
+          mocks: [
+            {
+              node_id: genericNodeData.id,
+              when_condition: {
+                type: "BINARY_EXPRESSION",
+                operator: "=",
+                lhs: {
+                  type: "WORKFLOW_INPUT",
+                  inputVariableId: "1",
+                },
+                rhs: {
+                  type: "CONSTANT_VALUE",
+                  value: {
+                    type: "STRING",
+                    value: "test-value",
+                  },
+                },
+              },
+              // then_outputs is intentionally undefined to test the fix
+            },
+          ],
+        },
+      ];
+
+      // WHEN we generate the sandbox file
+      const sandbox = codegen.workflowSandboxFile({
+        workflowContext: uniqueWorkflowContext,
+        sandboxInputs,
+      });
+
+      sandbox.write(writer);
+      const result = await writer.toStringFormatted();
+
+      // THEN it should not throw and should generate valid code
+      expect(result).toMatchSnapshot();
+      expect(result).toContain("MockNodeExecution");
+      // Verify that MockNodeExecution doesn't have then_outputs parameter
+      expect(result).toContain(
+        "MockNodeExecution(when_condition=Inputs.test_input.equals"
+      );
```

Comment:
DevinAI remove these two assertions so that the snapshot assertion is the only one